Wikipedia/Code/DiffHeaderView.swift

```diff
@@ -0,0 +1,89 @@
+import UIKit
+import CocoaLumberjackSwift
+
+final class DiffHeaderView: SetupView {
+
+    var viewModel: DiffHeaderViewModel?
+
+    lazy var horizontalStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .horizontal
+        stackView.alignment = .center
```

Comment:
Very cool seeing this come together! One thing I noticed is that as the header/subtitle text increases, the article image displays centered in the stack rather than pinned to the top (like how it functions on talk pages). You might want to test with a few different length strings to see how it looks in different conditions, but resolving this and pinning the image to the top may be as straightforward as updating this alignment to `.top`.

Reply:
Good catch, updated!